apps/wallet/src/ui/app/hooks/useLedgerNotification.tsx

```diff
@@ -0,0 +1,115 @@
+        );
+        const hasAcknowledged =
+            hasAcknowledgedVal === HAS_ACKNOWLEDGED_LEDGER_NOTIFICATION_VALUE;
+
+        if (isLedgerNotificationEnabled && !hasAcknowledged && !isMenuOpen) {
+            // If we don't have a timeout, the toast doesn't get rendered after initial render.
+            // We'll do this for now since we don't have the time to figure out what exactly is going on
+            setTimeout(() => {
+                toast.success(
+                    <div className="flex gap-2 items-center">
+                        <div className="shrink-0">
+                            <ButtonOrLink
+                                className="font-medium appearance-none border-0 cursor-pointer p-0 bg-transparent text-inherit"
+                                onClick={async () => {
+                                    trackEvent('LedgerNotification');
+                                    localStorage.setItem(
+                                        HAS_ACKNOWLEDGED_LEDGER_NOTIFICATION_KEY,
+                                        HAS_ACKNOWLEDGED_LEDGER_NOTIFICATION_VALUE
+                                    );
+                                    toast.remove(LEDGER_NOTIFICATION_TOAST_ID);
+
+                                    if (appType === AppType.popup) {
+                                        const { origin, pathname } =
+                                            window.location;
+                                        await Browser.tabs.create({
+                                            url: `${origin}/${pathname}#${connectLedgerModalUrl}`,
+                                        });
+                                        window.close();
+                                    } else {
+                                        navigate(connectLedgerModalUrl);
+                                    }
+                                }}
+                            >
+                                New! Tap to Connect your Ledger
+                            </ButtonOrLink>
+                        </div>
+                        <button
+                            className="w-full flex appearance-none border-0 p-0 bg-transparent cursor-pointer text-success-dark"
+                            onClick={() => {
+                                localStorage.setItem(
+                                    HAS_ACKNOWLEDGED_LEDGER_NOTIFICATION_KEY,
+                                    HAS_ACKNOWLEDGED_LEDGER_NOTIFICATION_VALUE
+                                );
+                                toast.dismiss(LEDGER_NOTIFICATION_TOAST_ID);
+                            }}
+                        >
+                            <div className="sr-only">Dismiss notification</div>
+                            <X12 />
+                        </button>
+                    </div>,
+                    {
+                        id: LEDGER_NOTIFICATION_TOAST_ID,
+                        className:
```

Comment:
this is super temporary and hacky, but we wanted the notification bar to have some custom styles that differ from the normal `success` toast